python/kfserving/kfserving/kfmodels/xgboost.py

```diff
@@ -16,34 +16,47 @@
 import xgboost as xgb
 from xgboost import XGBModel
 import os
-import numpy as np
-from typing import List, Dict
+from typing import Dict
 
 BOOSTER_FILE = "model.bst"
+DEFAULT_NTHREAD = 1
+
 
 class XGBoostModel(kfserving.KFModel):
-    def __init__(self, name: str, model_dir: str, nthread: int, booster: \
-        XGBModel = None):
+    def __init__(self, name: str, model_dir: str, nthread: int = DEFAULT_NTHREAD,
+                 booster: XGBModel = None):
```

Comment:
Should we just remove the `booster` argument? We should always load the model not init a XGBoostModel using an existing booster.